implementations/go/main.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/LordRahl90/botDevelopment/handlers"
```

Comment:
Hmm ... this might lead to some setup conflict. The different people that will build on your work might not have this as their own path. 🤔

Any ideas how we can go about this so that anyone can clone this and not have to change the path to get it to work?

Reply:
I have been worrying about this too... I will checkout how i can reference packages without using their fully qualified path and wont cause issue when am building too... Nice catch tho...

Reply:
@0sc I have been able to eliminate the liability... Please do lemme know ur opinion on the latest commit.

# Please do not merge yet
I am still getting the "no user found" issue.